### PR TITLE
docs: Update Token Type docs with new className parameter

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
- Update all the Token React types with className.

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: .changeset/happy-feet-do.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+- **chore**: Update Token Type docs with new className parameter. By @cpcramer #716
```
Comment (kyhyco) [inline .changeset/happy-feet-do.md:5]: nit
```suggestion
- **docs**: Update Token Type docs with new className parameter. By @cpcramer #716
```

Comment (cpcramer) [reply .changeset/happy-feet-do.md]: Updated!

File: site/docs/pages/identity/types.mdx [hunk 2]
```diff
@@ -9,8 +9,8 @@ description: Glossary of Types in Identity components & utilities.
 
 ```ts
 type AddressReact = {
-  address?: Address | null; // The Ethemreum address to render
-  className?: string; // className override for span element
+  address?: Address | null; // The Ethereum address to render
```
Comment (Zizzamia) [inline site/docs/pages/identity/types.mdx:12]: Love these corrections

Comment (cpcramer) [reply site/docs/pages/identity/types.mdx:12]: 😄

**Pull request merged**